devito/types/sparse.py

```diff
@@ -151,37 +151,6 @@ def _dist_scatter_mask(self, dmap=None):
         ret[self._sparse_position] = mask
         return tuple(ret)
 
-    def _dist_subfunc_scatter_mask(self, dmap=None):
-        """
-        This method is analogous to :meth:`_dist_scatter_mask`, although
-        the mask is now suitable to index into self's SubFunctions, rather
-        than into ``self.data``.
-        """
-        return self._dist_scatter_mask(dmap=dmap)[self._sparse_position]
```

Comment:
Note for reviewer: Slightly cheaper to just compute the mask and access sparse position than recopute it just to access the sparse position.